### ISSUE TITLE
Eplty package results in `package main` when `xmlstruct.WithFormatSource(false)`

### ISSUE THREAD

**Issue by meblum** [twpayne/go-xmlstruct#21]
https://github.com/twpayne/go-xmlstruct/commit/6cf7d4ce7169b58776696163621dc68da4aa38c4 introduced a regression when generating with
```
xmlstruct.WithPackageName(""),
xmlstruct.WithFormatSource(false),
```

The package declaration will now be applied  as` package main\n\n`

**Comment by meblum** [comment]
Output is also incorrect when running with

```
xmlstruct.WithHeader("do not edit"),
xmlstruct.WithPackageName(""),
xmlstruct.WithFormatSource(true),
```

**Comment by twpayne** [comment]
Thanks for spotting this. Fixed with b980a5747502b6e94a2b11da870403ef28007634.
